scrapinghub/client.py

```diff
@@ -282,21 +288,34 @@ class Spider(object):
 
     Usage::
 
-        >>> project.spiders.get('spider2')
+        >>> spider = project.spiders.get('spider1')
         <scrapinghub.client.Spider at 0x106ee3748>
```

Comment:
different from python interpreter output

```
>>> spider = project.spiders.get('localinfo')
>>> spider
<scrapinghub.client.Spider object at 0x10d6cb1d0>
```

I think you can simply remove `<scrapinghub.client.Spider at 0x106ee3748>`